app/main.py

```diff
@@ -40,6 +40,11 @@ def __init__(self):
         self._rabbit_port = settings.RABBIT_PORT
         self._default_user = settings.DEFAULT_USER
         self._default_pass = settings.DEFAULT_PASSWORD
+
+        self._corp_rabbit_port = settings.AWS_CORP_RABBITMQ_PORT
+        self._aws_user = settings.AWS_CORP_USER
+        self._aws_pass = settings.AWS_CORP_PASSWORD
```

Comment:
I think we should name the rabbitmq config to clearly show the source and destination queues i.e. eQ and SDX, rather than where the queues are hosted, something like:
```
SDX_GATEWAY_EQ_RABBITMQ_USER
SDX_GATEWAY_EQ_RABBITMQ_PASSWORD
SDX_GATEWAY_EQ_RABBITMQ_HOST
SDX_GATEWAY_EQ_RABBITMQ_HOST2
```
and 
```
SDX_GATEWAY_SDX_RABBITMQ_USER
SDX_GATEWAY_SDX_RABBITMQ_PASSWORD
SDX_GATEWAY_SDX_RABBITMQ_HOST
```

Reply:
Makes sense, though we need to update the gitlab config too